src/main/java/com/akademia/projectplanner/controller/HomeController.java

```diff
@@ -0,0 +1,25 @@
+package com.akademia.projectplanner.controller;
+
+import com.akademia.projectplanner.model.Task;
+import com.akademia.projectplanner.service.TaskService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import java.util.List;
+
+@Controller
+public class HomeController {
+
+    private TaskService taskService;
+
+    public HomeController(TaskService taskService) {
```

Comment:
Są dwa rodzaje projektów: takie co używają Lomboka i takie które nie. Trzymamy się jednej obranej konwencji, jeśli chcemy Lomboka to nie tworzymy już konstruktorów z "palca" tylko adnotacjami.